utam-tests/src/test/java/utam/examples/salesforce/RecordTestUtils.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) 2021, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: MIT
+ * For full license text, see the LICENSE file in the repo root
+ * or https://opensource.org/licenses/MIT
+ */
+package utam.examples.salesforce;
+
+import utam.global.pageobjects.RecordHomeFlexipage2;
+import utam.lightning.pageobjects.Datepicker;
+import utam.lightning.pageobjects.Input;
+import utam.lightning.pageobjects.Tabset;
+import utam.records.pageobjects.LwcHighlightsPanel;
+import utam.records.pageobjects.RecordLayoutItem;
+
+/**
+ * utilities for records testing
+ *
+ * @author elizaveta.ivanova
+ * @since Spring'22
+ */
+public class RecordTestUtils {
```

Comment:
@olivier-martin-sf  i found these methods are missing in artifact. for now will be utils